Tests/DataTable/RegistryTest.php

```diff
@@ -18,7 +19,7 @@ public function testRetrieveTableById()
         $registry = new DataTableRegistry(['test' => $table]);
         $this->assertEquals($table, $registry->retrieveByTableId('test'));
 
-        $this->setExpectedException('\BadMethodCallException');
+        $this->expectException('\BadMethodCallException');
         $registry->retrieveByTableId('undef');
     }
 }
```

Comment:
```suggestion
}

```
Please add a blank line before EOF

Reply:
Done.